tests/functional/reporting-issue-wizard-non-auth.js

```diff
@@ -110,7 +110,7 @@ registerSuite("Reporting with wizard", {
             // Click on "Desktop site instead of mobile site"
             document.querySelector("[for=problem_category-0]").click();
           })
-          .sleep(1000)
+          .sleep(1500)
```

Comment:
increasing timeouts is always a code smell that a test should probably be re-written. however, i'm a fan of getting things to pass and kicking the timeout can down the road a bit. :)

Reply:
yeah, we could probably add a function to check whether animation and scrolling to a step is finished instead of using timeouts.. I'll file a bug for that

Reply:
https://github.com/webcompat/webcompat.com/issues/3487